configs/openmrs_config/ampathforms/health_center_NCD_Screening.json

```diff
@@ -328,6 +328,30 @@
                             },
                             "type": "obs",
                             "validators": []
+                        },
+                        {
+                            "id": "HBA1c",
```

Comment:
@kazlaw I think we were supposed to add `Hb1AC` not `HBA1c` test as per the ticket requirement.